### ISSUE TITLE
jaraco.itertools.find_subseq() doctest failure

### ISSUE THREAD

**Issue by mgorny** [jaraco/jaraco.itertools#16]
This is on main @ b0481ccbdd33fc3474acf0af4b0f144e027e06cf.

```pytb
$ cat > jaraco/__init__.py <<-EOF
__path__ = __import__('pkgutil').extend_path(__path__, __name__)
EOF
$ python3.11 -m pytest --doctest-modules
========================================================= test session starts =========================================================
platform linux -- Python 3.11.3, pytest-7.3.1, pluggy-1.0.0
rootdir: /tmp/jaraco.itertools
configfile: pytest.ini
plugins: xprocess-0.22.2, forked-1.6.0, asyncio-0.21.0, pkgcore-0.12.21, xdist-3.3.1, anyio-3.7.0
asyncio: mode=Mode.STRICT
collected 38 items                                                                                                                    

jaraco/itertools.py ..................F...................                                                                      [100%]

============================================================== FAILURES ===============================================================
_______________________________________________ [doctest] jaraco.itertools.find_subseq ________________________________________________
1224         cand: iterable of items that must match
1225 
1226     Returns:
1227         The index where cand can be found in seq or None.
1228 
1229     >>> find_subseq([-1, 0, 1, 2], [1, 2])
1230     2
1231     >>> find_subseq([-1, 0, 1, 2], [0, 2])
1232     >>> find_subseq([-1, 0, 1, 2], [2, 1])
1233     >>> find_subseq([-1, 0, 1, 2], [])
Expected:
    Traceback (most recent call last):
    ValueError
Got:
    Traceback (most recent call last):
      File "/usr/lib/python3.11/doctest.py", line 1351, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest jaraco.itertools.find_subseq[3]>", line 1, in <module>
        find_subseq([-1, 0, 1, 2], [])
      File "/tmp/jaraco.itertools/jaraco/itertools.py", line 1241, in find_subseq
        match_indexes = more_itertools.locate(seq, check, window_size=len(cand))
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/usr/lib/python3.11/site-packages/more_itertools/more.py", line 2304, in locate
        raise ValueError('window size must be at least 1')
    ValueError: window size must be at least 1

/tmp/jaraco.itertools/jaraco/itertools.py:1233: DocTestFailure
======================================================= short test summary info =======================================================
FAILED jaraco/itertools.py::jaraco.itertools.find_subseq
==================================================== 1 failed, 37 passed in 0.82s =====================================================
```

**Comment by jaraco** [comment]
Thanks for the report. Fixed in 8730ea186d2d6bc149c9491f845200673f048f56.

**Comment by jaraco** [comment]
(let me know if you need this fix backported to Python 3.7)

**Comment by mgorny** [comment]
Thanks. No, we don't support anything older than 3.10 anymore.
